src/modules/collection/selectors.ts

```diff
@@ -30,10 +34,21 @@ export const getWalletCollections = createSelector<RootState, Collection[], stri
   (collections, address) => collections.filter(collection => address && isEqual(collection.owner, address))
 )
 
-export const getAuthorizedCollections = createSelector<RootState, Collection[], string | undefined, Collection[]>(
-  getCollections,
-  getAddress,
-  (collections, address) => collections.filter(collection => address && canSeeCollection(collection, address))
+export const getAuthorizedCollections = createSelector<
+  RootState,
+  Collection[],
+  string | undefined,
+  Record<string, ThirdParty>,
+  Collection[]
+>(getCollections, getAddress, getThirdParties, (collections, address, thirdParties) =>
+  collections.filter(collection => {
+    if (decodeURN(collection.urn!).type === URNType.COLLECTIONS_THIRDPARTY) {
```

Comment:
We can use collection/utils.ts `isThirdParty` method here

Reply:
Changed to `isThirdParty`!